code/__DEFINES/obj_flags.dm

```diff
@@ -34,6 +34,7 @@
 #define EYE_STAB				(1<<13) /// Item can be used to eyestab
 #define HAND_ITEM				(1<<14) /// If an item is just your hand (circled hand, slapper) and shouldn't block things like riding
 #define EXAMINE_SKIP			(1<<15) /// Makes the Examine proc not read out this item.
+#define XENOMORPH_HOLDABLE		(1<<16) /// A Xenomorph can hold this item.
```

Comment:
Add this new flag to `_globalvars/bitfields.dm` too.

Reply:
thanks